english/pokemon/pokemon-go.md

```diff
@@ -0,0 +1 @@
+Pokemon Go is a free-to-play location-based augmented reality mobile game developed by Niantic and published by The Pokémon Company. It was released in July 2016 for iOS and Android devices. 
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.